ophyd/signal.py

```diff
@@ -97,7 +100,48 @@ def put(self, value, *, timestamp=None, force=False, **kwargs):
         self._run_subs(sub_type=self.SUB_VALUE, old_value=old_value,
                        value=value, timestamp=self._timestamp)
 
-    set = put
+    def set(self, value, *, timeout=None, settle_time=None):
+        '''Set is like `put`, but is here for bluesky compatibility
+
+        Returns
+        -------
+        st : Status
+            This status object will be finished upon return in the
+            case of basic soft Signals
+        '''
+        def set_thread():
+            nonlocal timeout
+
+            if timeout is None:
+                timeout = 10
+                # TODO set_and_wait does not support a timeout of None
```

Comment:
Since `set_and_wait` polls and holds control in the meantime, I'm comfortable with this.